.github/workflows/rust.yml

```diff
@@ -10,6 +10,10 @@ env:
   CARGO_TERM_COLOR: always
   REDIS_RS_REDIS_JSON_PATH: "/tmp/librejson.so"
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

Comment:
Should we use `github.ref` instead of `github.head_ref`? If I'm reading this correctly, this wouldn't apply for multiple runs against `main`, for example.

Reply:
frankly, I don't know. I thought of using head_ref because it applies only to PRs, so I thought that would be a conservative first change. If you prefer `head`, I'll change.

Reply:
Yeah I think it's worth trying`${{ github.workflow }}-${{ github.ref }}` -- seems fine, and should hopefully kill redundant `main` jobs if we're merging multiple.